### ISSUE TITLE
TypeError: Cannot read property 'camelize' of undefined

### ISSUE THREAD

**Issue by iDuuck** [jhendley25/generator-dyno#27]
Don't have that much time to dig into this. Is anyone able to find the solution?

```
? What's the name of the project? (personal)
(!) #_ is deprecated. Require your own version of Lodash or underscore.string


/usr/local/lib/node_modules/yo/node_modules/inquirer/node_modules/rx/dist/rx.js:579
    throw e;
          ^
TypeError: Cannot read property 'camelize' of undefined
    at Object.prompts.filter (/usr/local/lib/node_modules/generator-dyno/app/index.js:29:22)
    at Object.utils.runAsync (/usr/local/lib/node_modules/yo/node_modules/inquirer/lib/utils/utils.js:29:22)
    at Prompt.filter (/usr/local/lib/node_modules/yo/node_modules/inquirer/lib/prompts/base.js:184:9)
    at Prompt.onEnd (/usr/local/lib/node_modules/yo/node_modules/inquirer/lib/prompts/input.js:81:8)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.next (/usr/local/lib/node_modules/yo/node_modules/inquirer/node_modules/rx/dist/rx.js:1962:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/lib/node_modules/yo/node_modules/inquirer/node_modules/rx/dist/rx.js:1894:35)
    at AnonymousObserver.tryCatcher (/usr/local/lib/node_modules/yo/node_modules/inquirer/node_modules/rx/dist/rx.js:567:29)
    at AutoDetachObserverPrototype.next (/usr/local/lib/node_modules/yo/node_modules/inquirer/node_modules/rx/dist/rx.js:5018:51)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onNext (/usr/local/lib/node_modules/yo/node_modules/inquirer/node_modules/rx/dist/rx.js:1894:35)
    at AnonymousObserver._onNext (/usr/local/lib/node_modules/yo/node_modules/inquirer/node_modules/rx/dist/rx.js:4826:13)
```

Also. Switch to `underscore.string`, if this is related to an issue with `generator-dyno`

**Comment by jhendley25** [comment]
@iDuuck I can't duplcate that issue. What version of node and npm are you on?

**Comment by iDuuck** [comment]
```
Nickbook(master*) $ npm -v
2.7.4
Nickbook(master*) $ node -v
v0.12.2
```

**Comment by iDuuck** [comment]
I found the issue and fixed it on 7dd8f6e. I think it was because of an updated version of `yeoman-generator` that deprecated the `underscore`-thingy. Just added `underscore.string` and replaced the 'missing' methods.

**Comment by jhendley25** [comment]
Awesome. Thanks :+1:
